test.php

```diff
@@ -2,107 +2,101 @@
 
 namespace App\Importer;
 
-use App\Entity\ClientImporter;
 use App\Entity\FieldConfiguration;
-use App\Model\ImportCriteria;
-use App\Model\SourceField;
 use DateTime;
 use Doctrine\DBAL\Connection;
 use Doctrine\DBAL\DriverManager;
 use Doctrine\DBAL\Query\QueryBuilder;
-use Exception;
 use Google\Cloud\BigQuery\BigQueryClient;
 use MathParser\Exceptions\DivisionByZeroException;
 use MathParser\Interpreting\Evaluator;
-use MathParser\Parsing\Nodes\FunctionNode;
 use MathParser\StdMathParser;
 
 class BaseBigQueryImporter extends BaseImporter
 {
-   private ?Connection $connection = null;
-   private ?BigQueryClient $client = null;
+    private ?Connection $connection = null;
+    private ?BigQueryClient $client = null;
 
-   protected string $tablePrefix = '';
-   protected string $clientKeyField = 'companyName';
-   protected string $dateField = 'date';
-   protected ?string $dateFieldSelect = 'date';
-   protected string $asinField = 'asin';
-   protected ?string $asinFieldSelect = 'asin';
-   protected string $rootTableAlias = 'root';
+    protected string $tablePrefix = '';
+    protected string $clientKeyField = 'companyName';
+    protected string $dateField = 'date';
+    protected string $dateFieldSelect = 'date';
+    protected string $asinField = 'asin';
+    protected string $asinFieldSelect = 'asin';
+    protected string $rootTableAlias = 'root';
 
-   /**
-    * @throws Exception
-    */
-   public function getConnection(): Connection
-   {
-       if (null != $this->connection) {
-           return $this->connection;
-       }
-       /*Empty connection to use doctrine to build queries and then use them in big query*/
-       $this->connection = DriverManager::getConnection([
-//            'url' => $_ENV['DATABASE_URL'],
-           'url' => getenv('DATABASE_URL'),
-           'driver' => 'pdo_pgsql',
-       ]);
+    /**
+     * @return Connection
+     */
+    public function getConnection(): Connection
+    {
+        if (null != $this->connection) {
+            return $this->connection;
+        }
 
-       return $this->connection;
-   }
+        $this->connection = DriverManager::getConnection([
+            'host' => getenv('DATABASE_URL'),
```

Comment:
Probably the problem is here because the connection not admits URL just the `host`option according to this [documentation](https://www.doctrine-project.org/projects/doctrine-dbal/en/latest/reference/configuration.html)